test/serializable_resource_test.rb

```diff
@@ -23,5 +23,13 @@ def test_serializable_resource_delegates_as_json_to_the_adapter
       options = nil
       assert_equal @adapter.as_json(options), @serializable_resource.as_json(options)
     end
+
+    def test_use_adapter_with_adapter_option
+      assert_equal ActiveModel::SerializableResource.new(@resource, { adapter: 'json' }).use_adapter?, true
```

Comment:
oh, missed the expectation here.  it's `assert_equal(expected, actual)`. But in this case, since it's boolean, it should just be `assert` and in the next test `refute`